soccer/src/soccer/planning/planning_params.cpp

```diff
@@ -62,7 +62,7 @@ DEFINE_NS_FLOAT64(
 //
 // High number indicates that it will always try to choose a point nearest
 // to the current robot position
-DEFINE_NS_FLOAT64(kPlanningParamModule, collect, ball_speed_approach_direction_cutoff, 0.1,
+DEFINE_NS_FLOAT64(kPlanningParamModule, collect, ball_speed_approach_direction_cutoff, 1.0,
```

Comment:
the relevant lines of code in the collect planner [are here](https://github.com/RoboJackets/robocup-software/blob/2a4bacfd1b3f4a01c4fcdcedc6c9060e71f436dd/soccer/src/soccer/planning/planner/collect_planner.cpp#L88-L96). you can see that when the ball speed is < this thresh, the robot will try to go directly at it, while if the speed is > this thresh, the robot will try to serve as a backstop.

what I noticed in my testing is that the robot didn't backstop successfully very often, so I tuned a bunch of params including this and the accel_scale ones below. those changes combined actually led to less bouncing, but it's hard to isolate any of them. if you want, you can make a card that's like "tune capture" but I think a) that's not in the scope of this PR and b) that's not easy to do when gameplay-controlled passing has some timing bugs. I think once ball placement is up (with passing) it will be easier to see, so I decided to wait on tuning this and any other skills